appinventor/components/src/com/google/appinventor/components/runtime/util/TextViewUtil.java

```diff
@@ -175,7 +176,12 @@ public static String getText(TextView textview) {
    * @param text  new text to be shown
    */
   public static void setTextHTML(TextView textview, String text) {
-    textview.setText(Html.fromHtml(text));
+    if (android.os.Build.VERSION.SDK_INT >= 24 {
```

Comment:
You are also importing this so you don't need to specify the fully-qualified name here.

Reply:
Apologies Conor I completely forgot how to deal with this import on my previous extension as well, will fix right away!